com.francescogaglione.cosmicmoney.json

```diff
@@ -0,0 +1,57 @@
+{
+  "id": "com.francescogaglione.cosmicmoney",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "cosmic_money",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--talk-name=com.system76.CosmicSettingsDaemon",
+    "--filesystem=xdg-config/cosmic:rw",
```

Comment:
why is rw needed?

Reply:
THis needs an exception, did you run the linter like the docs say?

Reply:
https://docs.flathub.org/docs/for-app-authors/linter#exceptions

Reply:
sorry it was an oversight